### PR TITLE
Extends Hass.io installation manual for generic Linux

### PR DESCRIPTION
**Pull request by Kanga-Who**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

<a href="https://gitpod.io/#https://github.com/home-assistant/home-assistant.io/pull/9972"></a>

### CODE REVIEW

File: source/hassio/installation.markdown [hunk 1]
```diff
@@ -152,12 +152,18 @@ apt-get install -y apparmor-utils apt-transport-https avahi-daemon ca-certificat
 curl -fsSL get.docker.com | sh
 ```
 
-And to install Hass.io the one below. That one is used also for other distributions.
+The below script works for generic installs on x86/x64 machines, running an OS such as Ubuntu.
```
Comment (frenck) [inline source/hassio/installation.markdown]: I think the original one was better, especially this is not true. 
Hass.io is not limited to x86/x64 machines. Please restore the original line.

File: source/hassio/installation.markdown [hunk 2]
```diff
@@ -152,12 +152,18 @@ apt-get install -y apparmor-utils apt-transport-https avahi-daemon ca-certificat
...
 ```
 
-And to install Hass.io the one below. That one is used also for other distributions.
+The below script works for generic installs on x86/x64 machines, running an OS such as Ubuntu. 
 
 ```bash
 curl -sL "https://raw.githubusercontent.com/home-assistant/hassio-installer/master/hassio_install.sh" | bash -s
 ```
 
+Some installation types require flags to identify the computer type, for example, when using a Raspberry Pi 3, the flag `-- -m raspberrypi3` is required. The install script would then look like this.
```
Comment (frenck) [inline source/hassio/installation.markdown]: Very useful section, maybe extend this a bit with the possible machine types?

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by Kanga-Who** [comment]
Thanks, @frenck.

I have made some changes and welcome your feedback.

**Review by frenck — APPROVED** [review]
Thanks, @Kanga-Who! 👍

**Comment by Kanga-Who** [comment]
Great! Thanks @frenck.

**Pull request merged**